intents/util/constants.py

```diff
@@ -33,3 +33,9 @@
 LOOKING_AT_THE_TAILS_FLOAT_THRESHOLD = .7 
 # type is angle, as cosine similarity is used
 LOOKING_AT_THE_TAILS_STRING_THRESHOLD = 20
+
+# threshold value of simpson's-paradox.
+SP_THRESHOLD = 75
```

Comment:
Give it a more descriptive name. Such as SIMPSONS_PARADOX_DOMINANT_PERCENT_THRESHOLD

Reply:
fixed